lib/angular-evaporate.js

```diff
@@ -113,6 +113,16 @@
             this.contentType = this.file.type || 'binary/octet-stream';
             this.$url = ae.$url + this.name;
 
+            function readableFileSize(size) {
```

Comment:
sorry, I don't get why it will be recreated with every `forEach`. It's declared before that block.